docs/codeql/writing-codeql-queries/introduction-to-ql.rst

```diff
@@ -113,8 +107,6 @@ To simplify the query, we can introduce a class ``SmallInt`` representing the in
    where x.square() + y.square() = z.square()
    select x, y, z
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/2101340747/>`__
-
 Example CodeQL queries
 ----------------------
 
```

Comment:
I think that the following examples will need to VS Code set up.

Reply:
Yep, from here on you need a VS Code set up.

Reply:
Good callout! ✨ I added the paragraph on line 119 to cover this, and incorporated some other feedback you left there too.